prefs/prefs.go

```diff
@@ -0,0 +1,56 @@
+package prefs
+
+import (
+	"pkg.re/essentialkaos/ek.v8/knf"
+	"pkg.re/essentialkaos/ek.v8/fmtc"
+	"os"
```

Comment:
I have trying to organize imports somehow, but IDEA (Gogland) sometimes changes an order of import automatically. Right now it looks fine.